third_party/llvm/include/llvm/Demangle/README.txt

```diff
@@ -0,0 +1,61 @@
+Itanium Name Demangler Library
```

Comment:
This Readme discusses the organization of the LLVM code, referencing certain features of that particular library.  It may be better to omit this so as to not be confusing, although removing it is also confusing because the implementation references this readme.  🤷